src/Schema/State.php

```diff
@@ -108,6 +108,10 @@ public function setPrimaryKeys(array $columns): void
 
     /**
      * Method combines primary keys with primary keys automatically calculated based on registered columns.
+     *
+     * @return string[]
+     *
+     * @psalm-return list<non-empty-string>
```

Comment:
```suggestion
     * @return list<non-empty-string>
```

You can skip the `psalm` prefix in this cases